client/client.go

```diff
@@ -0,0 +1,130 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/Layr-Labs/eigenda-proxy/common"
+	"github.com/Layr-Labs/eigenda-proxy/eigenda"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+const (
+	// NOTE: this will need to be updated as plasma's implementation changes
+	decodingOffset = 2
+)
+
+// TODO: Add support for custom http client option
+type Config struct {
+	URL string
+}
+
+// ProxyClient is an interface for communicating with the EigenDA proxy server
+type ProxyClient interface {
+	Health() error
```

Comment:
Is this part of the OP plasma interface? I figured this endpoint was for docker healthchecks.